addons/hr_timesheet/views/hr_timesheet_views.xml

```diff
@@ -188,6 +210,7 @@
             <field name="view_mode">tree,form</field>
             <field name="domain">[('project_id', '!=', False), ('user_id', '=', uid)]</field>
             <field name="context">{
+                "timer": True,
```

Comment:
1/ Try to remove the context key
2/ Try to remove it again, if 1/ failed.
3/ Rename it into something more explicit and easy to grep than 'timer'. (Currently 715 matches in master). For example, 'hr_timesheet_display_timer' (if this is what it does).

Reply:
I use this to display the timer only in the 'My timesheet' page, because the 'My timesheet' and 'All timesheets' pages use the same views. 
I renamed this magic context to "my_timesheet_display_timer".